eth2/fork_choice/src/lib.rs

```diff
@@ -44,13 +44,17 @@ extern crate types;
 
 pub mod longest_chain;
 pub mod optimised_lmd_ghost;
-pub mod protolambda_lmd_ghost;
+//pub mod protolambda_lmd_ghost;
 pub mod slow_lmd_ghost;
 
 use db::stores::BeaconBlockAtSlotError;
 use db::DBError;
 use types::{BeaconBlock, Hash256};
 
+// export the main structs
```

Comment:
Total nit-pick, but not sure this is necessary

Reply:
I think it's nice because when importing them you have this long thing like, 
use fork_choice::optimised_lmd_ghost::OptimisedLMDGhost
And if you want to change it you equally have to write out the other middle name. Think it's nicer like:
use fork_choice::OptimisedLMDGhost.

Reply:
I was just meaning the comment, agree with the short imports.